generated/ruby/grpc-google-iam-v1/grpc-google-iam-v1.gemspec

```diff
@@ -0,0 +1,26 @@
+# -*- ruby -*-
+# encoding: utf-8
+
+Gem::Specification.new do |s|
+  s.name          = 'grpc-google-iam-v1'
+  s.version       = '0.6.8'
```

Comment:
Let's use the 0.x versions that we're currently using for the individual languages. We don't want to declare 1.0 before we have to.